src/pages/about/research.js

```diff
@@ -27,9 +27,6 @@ const ResearchPage = ({ data }) => {
               paddingLeft: "1rem",
             }}
           >
-            <Paragraph>
-              Published on {date} in <em>{location}</em>
-            </Paragraph>
             <Paragraph>
               {title ===
```

Comment:
is this ideal? there another way to accomplish this other than this very specific hack? what happens when there's another one-off publication like this?